### ISSUE TITLE
the problem of en-de

### ISSUE THREAD

**Issue by travel-go** [facebookresearch/fairseq-py#124]
Hello,When I run the en-de model, I find that my bleu is lower than in the paper, then I would like to ask if your validation set is newtest2014?

**Comment by myleott** [comment]
Do you mean the pretrained models? This is what I see:

```
python generate.py \
  data/data-bin/wmt14.en-de.newstest2014/ \
  --path wmt14.en-de.fconv-py/model.pt \
  --gen-subset test --remove-bpe
(...)
| Generate test with beam=5: BLEU4 = 25.71, 56.8/31.2/19.5/12.6 (BP=1.000, ratio=1.022, syslen=60585, reflen=59297)
```

The paper reports single-model performance of 25.16 for this model on newstest2014 (Table 1).

**Comment by travel-go** [comment]
Sorry, I was wrong,I want to ask if your development set on en-de is newstest2013?

**Comment by michaelauli** [comment]
The validation set is held out from the training data, we do not currently provide newstest2013.

**Comment by travel-go** [comment]
thank you for your replay.I follow your en-de model setting,the wmt14‘s En-De data is 4468840 sentences and newstest2013(3000 sentences) is ued as a development set.The BLEU scores on newstest2014 is 23.78.So I wonder if something is worry。

**Comment by ShilinHe** [comment]
Did you figure this out? I find that the newstest2014 (2737) provided by Stanford does not match the original one (3003 sentences) released by WMT14 En-De.
